app/controllers/chapters_controller.rb

```diff
@@ -194,38 +195,129 @@ def chapter_3
   def chapter_4
     @data = @chapters_data[3]
 
-
-    # TODO - Need new data for [Map of ecoregion coverage: marine and terrestrial on the same map. Toggle between PAs only and PAs + OECMs]  map here!
+    @map = {
```

Comment:
Looking at this, part of me wants to refactor it into the Chapters presenter. Can I do it as part of this PR?